src/test/java/th/ac/kmitl/science/comsci/example/models/ProfileTest.java

```diff
@@ -0,0 +1,24 @@
+package th.ac.kmitl.science.comsci.example.models;
+
+import org.junit.Test;
+
+public class ProfileTest extends Profile {
```

Comment:
why ProfileTest need to extend Profile krub?

Reply:
Because Profile is an abstract class cannot create objects for test.

Reply:
@demonsker อืม งั้นพี่ว่าเอา ProfileTest ออก แล้วใส่ //TODO ไว้แทนดีกว่า ว่าเวลาเราทำ Seller หรือ Buyer (implementation ของ Abstract class นี้) ค่อยเขียน test ตัวนี้

แต่.... มันต้องเป็น Abstract class จริงรึป่าวนี่สิ?
ลองตรวจสอบให้พี่หน่อยสิ ว่า Seller กับ Buyer data structure ตรงกันรึป่าว?
ถ้าตรงกัน ก็ไม่ต้องมี class Seller กับ Buyer ถูกมะ แค่ seller และ buyer เป็น object ที่ได้จาก class Profile
(แต่ถ้าเป็นงั้น ควรเปลี่ยนชื่อ Profile เป็นอย่างอื่นครับ เช่น Company จะสื่อกว่า เพราะ seller และ buyer ก็เป็นบริษัทด้วยกันทั้งนั้น ซึ่งกันและกัน)

Reply:
ทั้ง Seller และ Buyer มี structure เดียวกันครับ ถ้างั้นผมจะเปลี่ยนเป็น Company นะครับ